x-pack/platform/plugins/shared/streams/server/routes/content/route.ts

```diff
@@ -155,7 +186,40 @@ const importContentRoute = createServerRoute({
   },
 });
 
+const previewContentRoute = createServerRoute({
+  endpoint: 'POST /api/streams/{name}/content/preview 2023-10-31',
```

Comment:
should this be an internal endpoint? A proper API consumer wouldn't have the issue with zip.js, right?

Reply:
Yes this endpoint really only exists for our UI to read the content without introducing a new library so internal sounds good